.ebextensions/cronjob.config

```diff
@@ -248,6 +248,28 @@ files:
           0 0 1 * * root /bin/bash -l -c 'cd /var/app/current && RAILS_ENV=production bin/run_as_webapp bundle exec rake alert:delete_benchmark_runs >> log/jobs.log 2>&1'
 
 
+    #
+    # Send Delayed Job memory usage metrics to AWS Cloudwatch
+    #
+    "/etc/cron.d/send-delayed-job-memory-usage-metrics-to-cloudwatch":
+        mode: "000644"
+        owner: root
+        group: root
+        content: |
+          */5 * * * * root aws cloudwatch put-metric-data --metric-name MemoryUsage --namespace DelayedJob --value=$(./current/script/delayed_job_memory_usage.sh) --unit Kilobytes --region=eu-west-2 --dimensions InstanceId=$(ec2-metadata --instance-id | cut -d " " -f 2)
+
+
+    #
+    # Send Delayed Job process metrics to AWS Cloudwatch
+    #
+    "/etc/cron.d/send-delayed-job-process-metrics-to-cloudwatch":
+        mode: "000644"
+        owner: root
+        group: root
+        content: |
+          */5 * * * * root aws cloudwatch put-metric-data --metric-name ProcessCount --namespace DelayedJob --value ./script/delayed_job_number_of_work_process.sh --unit Count --dimensions ec2-metadata --instance-id
```

Comment:
The parameters are different to the other job. `--value` is assigned using different syntax, there's no region, and there instance id doesn't look like its set correctly? Is this a mistake or is there a reason for the differences?

Reply:
@ldodds yep, that was a mistake, sorry about that.  fixed now in the pr and tested on the `test-3-4-9`.  this should be ok to go now.